testing/moduletest.py

```diff
@@ -0,0 +1,30 @@
+##Airtable API/.env test 
+import os
+from dotenv import load_dotenv, dotenv_values
+from flask import Flask, render_template
+import requests as rq
+
+
+load_dotenv() 
+PAT = (os.getenv("PAT")) #-> str: #initalizes personal access token
+BASE = "appr8SepSgx9SP6ir" #zummobikes base id
+TBL = "tblXBR1YTXYvGhwWJ" #zummobikes table id
```

Comment:
Maybe specify that these are for airtable in the variable names?  We might be using stripe later on